### ISSUE TITLE
Create MailChimp newsletter for Link Roundups

### ISSUE THREAD

**Issue by MirandaEcho** [INN/umbrella-aspenj#6]
- [x] Create campaign, audiences, & basic template in MailChimp for daily link roundup for his "Bucket" section of the site
- [x] Update signup form to include checkboxes for daily roundup, weekly stories, and monthly updates

**Comment by joshdarby** [comment]
>  Create campaign, audiences, & basic template in MailChimp for daily link roundup for his "Bucket" section of the site

Mailchimp is yelling at me that they've hit their allotted number of audiences allowed for their plan (5).

**Comment by joshdarby** [comment]
> basic template in MailChimp for daily link roundup for his "Bucket" section of the site

@MirandaEcho Do we know...

- [x] what section of his site is the "Bucket" section
- [x] does he want to automate this like the other newsletter
- [x] if he does want to automate it, do they already have a feed set up for posts with this specific tag besides https://www.aspenjournalism.org/?feed=mailchimp?

**Comment by joshdarby** [comment]
> Update signup form to include checkboxes for daily roundup, weekly stories, and monthly updates

Do we know which list/audience is for monthly updates?

**Comment by joshdarby** [comment]
Based on https://github.com/INN/link-roundups/blob/master/docs/mailchimp.md, it looks like we'll need to make sure he knows that he will need to create actual roundup posts as opposed to just saving links.

Currently there are 11,xxx saved links but 0 roundups.

**Comment by joshdarby** [comment]
I think this is ready for testing by us/him.

I have:
- created new `Bucket` audience
- created new `Bucket Roundup` template based on the existing newsletter template and updated it with the relevant LR tags detailed here https://github.com/INN/link-roundups/blob/master/docs/mailchimp.md
- added a new checkbox field to the form @benlk created for #7 (found here: http://staging.overrated-board.flywheelsites.com/subscribe/) that allows users to select which lists/audiences to subscribe to. It is set up as:
   - `Daily Roundup` : `Bucket` audience
   - `Weekly Roundup` : `Master list` audience, `Top Stories (weekly email)` list/group
   - `Monthly Update` : `Master list` audience, `The Source (monthly e-newsletter)` list/group

**Comment by joshdarby** [comment]
Campaign in MC: https://us5.admin.mailchimp.com/campaigns/edit?id=3858541

@benlk Since this will be automated via Link Roundups, does this need to be an RSS email campaign in MC or just a regular email campaign?

**Comment by benlk** [comment]
I think it's a regular campaign, but I may be wrong. Either way, we should document that in Link Roundups.

**Comment by joshdarby** [comment]
@MirandaEcho Update on this since we're at about time:

Everything is configured. However, Link Roundups isn't populating the automated Mailchimp email the way it should be.

@benlk and I took a few minutes after Scrum to discuss it but can't find an obvious error/issue with it so far. Exporting the logs from Flywheel didn't prove to be helpful and WP Debug isn't showing any issues either.

**Comment by joshdarby** [comment]
Spoke with Mailchimp support and they mentioned the most likely reason this isn't working is because LR is using V2 of their API which is no longer supported/maintained 😬

**Comment by joshdarby** [comment]
Now that this magically works:

@MirandaEcho Do we want to modify anything on the template, like order of items displayed or anything like that?

![Screen Shot 2020-04-22 at 12 09 19 PM](https://user-images.githubusercontent.com/18353636/80005996-4c64fd80-8492-11ea-8784-f1457dc3ec2a.png)

**Comment by joshdarby** [comment]
- [x] remove saved links from main mailchimp feed
- [x] switch language to match "the bucket" & subtext from the site. News of interest to Aspenites
- [x] remove "weekly" language at the top
- [x] make RSS feed that pulls in most recent saved links from 12pm-12pm
- [x] set up email template to automate sending from that feed

**Comment by joshdarby** [comment]
So I have the automated email all set up in Mailchimp and it seems to be working as expected. However, quick question for @benlk:

I found this in their [docs](https://mailchimp.com/help/troubleshooting-rss-in-campaigns/):

> We pull in the date for any of these tags, in this order: 'pubDate', 'pubdate', 'published', 'created', 'updated', 'date.' If you set your campaign to daily, it will send posts from the last 24 hours before it was triggered;

Do you think we'd be better off removing the whole time conditional from the feed and just let Mailchimp handle it? Seems like that would be more future proof in case any of the timezone settings in WP or the server side get messed with.

**Comment by benlk** [comment]
Yes, that would be more future-proof. But if we're doing that, we should remove the `-1` loading of all saved links, and instead replace it with a smaller number, like `100`.
